test/serverless-v2/RelaySubscriptions.ts

```diff
@@ -561,7 +636,7 @@ describe("RelaySubscriptions - Job Subscription Response", function () {
 
         let jobSubsParams = {
             startTime: 0,
-            maxGasPrice: 10,
+            maxGasPrice: (await signers[0].provider?.getFeeData())?.gasPrice || parseUnits("1", 9),
```

Comment:
why is this required?

Reply:
The test was failing with the low value of maxGasPrice. Initially, it was working fine, but I think after the deps were updated recently(by Roshan), it popped up. So updated it to a higher value.